status-EIPs.md

```diff
@@ -220,6 +221,14 @@ Used for: Security (dont reuse wallet key) and user experience (dont request key
 Related: https://github.com/status-im/status-react/issues/9088 https://github.com/status-im/status-react/pull/9096  
 Sourcecode: https://github.com/status-im/status-react/blob/develop/src/status_im/constants.cljs#L242  
 
+### EIP1459 - Node Discovery via DNS
+
+Support: -
+Reference: https://eips.ethereum.org/EIPS/eip-1459
+Description: Allows the storing and retreving of nodes through merkle trees stored in TXT records of a domain.
+Used for: Finding Waku nodes.
```

Comment:
I'd clarify this to say:
- used for finding bootstrap nodes

As well as currently:
- to find Waku nodes

Then once we get to integrate more proper support for Discovery v5 (or similar), we can adjust spec

Reply:
But we only support Waku nodes in DNS for now. If one puts a bootstrap node in there, they will be ignored. In the future, we can change it and even have a mix of nodes. This will be handled fine as long as a node is a valid enode address.